docs/src/terminology.md

```diff
@@ -138,6 +138,10 @@ The role of a [validator](terminology.md#validator) when it is appending [entrie
 
 A sequence of [validator](terminology.md#validator) [public keys](terminology.md#public-key). The cluster uses the leader schedule to determine which validator is the [leader](terminology.md#leader) at any moment in time.
 
+## leader schedule offset
+
+To ensure that a [leader schedule](terminology.md#leader-schedule) is confirmed by a supermajority of validators before it takes effect, it must be placed in a block far enough in advance of the epoch boundary.
```

Comment:
This isn't a definition for leader schedule offset. It just explains how the offset is used.